fix: quickstart filename capitalisation not clear (#6935)
EddieHubCommunity/LinkFree: pages/docs/quickstart.mdx

![Create new file](https://user-images.githubusercontent.com/51131670/210201815-9c56ca28-19e8-4ef9-9934-6014d5262624.png)

7. In the file path at the top, add `data/` followed by your GitHub username, exactly as it appears including capitalisation, and then append `.json` to it.

> 

Reviewer: ```suggestion
```

![Create new file](https://user-images.githubusercontent.com/51131670/210201815-9c56ca28-19e8-4ef9-9934-6014d5262624.png)

> 7. In the file path at the top, add `data/` followed by your GitHub username and then append `.json` to it.
> 
> Please take care of the capitalization of your username.

Reviewer: ```suggestion
7. In the file path at the top, add `data/` followed by your GitHub username, exactly as it appears including capitalisation, and then append `.json` to it.

```
